<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="iterationsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Iterations:</string>
   </property>
  </widget>
  <widget class="QLabel" name="iterations">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="gridSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 x 0 x 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Grid Size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>161</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delta P:</string>
   </property>
  </widget>
  <widget class="QLabel" name="deltaP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>101</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>151</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MLUPS (avg.):</string>
   </property>
  </widget>
  <widget class="QLabel" name="performance">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>171</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="expendedTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>101</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>81</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Avg. P:</string>
   </property>
  </widget>
  <widget class="QLabel" name="averageP">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>101</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="disableCharts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Disable Charts</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="multicoreSupport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Multicore Support</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>171</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Perfomance Tweaks:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>650</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Render Update:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="renderUpdate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="remainingTime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>101</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>171</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remaining:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>650</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Exec. Burst:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="executionBurst">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>670</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="infoX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="infoY">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="infoZ">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="velocityX">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Velocity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Density:</string>
   </property>
  </widget>
  <widget class="QLabel" name="density">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vorticity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vorticityX">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Stream:</string>
   </property>
  </widget>
  <widget class="QLabel" name="stream">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="velocityY">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="velocityZ">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vorticityY">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vorticityZ">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="infoZ2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="infoY2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>530</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="ImprovedLineEdit" name="infoX2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImprovedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>improvedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>infoX</tabstop>
  <tabstop>infoY</tabstop>
  <tabstop>infoZ</tabstop>
  <tabstop>infoX2</tabstop>
  <tabstop>infoY2</tabstop>
  <tabstop>infoZ2</tabstop>
  <tabstop>multicoreSupport</tabstop>
  <tabstop>disableCharts</tabstop>
  <tabstop>renderUpdate</tabstop>
  <tabstop>executionBurst</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
